### ISSUE TITLE
Alt Text

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/AdminImages#2]
Alt tags not being honored. 

(Already discussed -- just putting it here to keep it on the list.)

**Comment by NedHenry** [comment]
I have now implemented the planned shortcodes and view helpers which take advantage of the alt text and title text fields. I added brief instructions on how to use these. Please let me know if you think the functionality or instructions should be changed.
